src/gui_common/MainMenu.cs

```diff
@@ -118,9 +118,6 @@ private void RunMenuSetup()
         options = GetNode<OptionsMenu>("OptionsMenu");
         saves = GetNode<SaveManagerGUI>("SaveManagerGUI");
 
-        // Load settings
-        options.SetSettingsFrom(Settings.Instance);
```

Comment:
This line is here in order to ensure that the Settings object is loaded from json.
I think it makes things a bit clearer that options is forced to exist here.

Reply:
You mean referencing it so it is forced to be statically constructed? Cause that method call essentially synchronizes the state of the gui controls to what the loaded settings are(SetSettingsFrom). I thought the whole line wasn't very obvious what it is doing, personally. Instead I switched it to sync them in _ready. I also started using that method to sync it when you hit reset or anything else that adjusts the settings without you manipulating the controls.

Reply:
With the previous code, the settings object was created the first time Settings.Instance was referenced.
Similarly to what this line does: https://github.com/Revolutionary-Games/Thrive/blob/e51908354b8ba6869db0014617cf09aa7d85d0d4/src/microbe_stage/MicrobeStage.cs#L166

So if you moved things that way, I'd still like to have at least
```
// Load settings
Settings.Instance;
```
here to fulfill that purpose that the settings are loaded here (I don't want the settings object to be loaded too early, just in case in the future some mods or early autoloads want to customize things)

If that code complains then checking against null would be fine here as well and reporting an error if it is null.

Reply:
Oh, then yeah if it's just about forcing the static to be loaded then that's fine. I can add that line in.